### ISSUE TITLE
Support for TMSCHEMA tables to test SSAS tabular model

### ISSUE THREAD

**Issue by shiv24saxena** [Seddryck/NBi#616]
Currently Tabular model metadata is also getting verified against MDSCHEMA tables which are mainly meant for multidimensional model.
Tabular model metadata information is stored in TMSCHEMA tables which should be used to test the model.

Thanks

**Comment by Seddryck** [comment]
You're right but could you be a bit more specific on what's going not well with the current implementation?

**Comment by shiv24saxena** [comment]
Thanks for responding. Few scenarios where current implementation will not work:
1) In Tabular model to get mapping between Technical column name (Database field example FIRST_NM) and business column name (example - First Name), we need use TMSCHEMA_COLUMNS. Same information can't be retrieved by DBSCEHMA_COLUMNS.
2) To get perspective details in case of tabular model, we need use TMSCHEMA_PERSPECTIVE_TABLES and TMSCEHMA_PERSPECTIVE_COLUMNS table. Same can't be retrieved from MDSCHEMA_DIMENSIONS which I believe is getting used currently,
Please let me know if any other information/scenario is required.
